src/main/java/uk/co/evoco/webdriver/configuration/WebDriverConfig.java

```diff
@@ -110,15 +106,13 @@ public void setTestConfig(JsonNode testConfig) {
     }
 
     /**
-     *
      * @return boolean configuring headless running
      */
     public boolean isHeadless() {
         return isHeadless;
     }
 
     /**
-     *
```

Comment:
These need adding back in - javadocs convention is to have a blank line first which is where the description goes